src/main/java/net/glowstone/net/message/play/game/BlockBreakAnimationMessage.java

```diff
@@ -0,0 +1,10 @@
+package net.glowstone.net.message.play.game;
+
+import com.flowpowered.network.Message;
+import lombok.Data;
+
+@Data
+public class BlockBreakAnimationMessage implements Message {
+
+    private final int entityID, x, y, z, destroyStage;
```

Comment:
For the sake of continuity with other entity Message classes, change `entityID` to `id`.